cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/prometheus/PrometheusAdapterTest.java

```diff
@@ -0,0 +1,234 @@
+            + "                        1603301400,\n"
+            + "                        \"1024\"\n"
+            + "                    ],\n"
+            + "                    [\n"
+            + "                        1603301430,\n"
+            + "                        \"2048\"\n"
+            + "                    ]\n"
+            + "                ]\n"
+            + "            },\n"
+            + "            {\n"
+            + "                \"metric\": {\n"
+            + "                    \"__name__\": \"kafka_server_BrokerTopicMetrics_OneMinuteRate\",\n"
+            + "                    \"instance\": \"b-2.test-cluster.org:11001\",\n"
+            + "                    \"job\": \"jmx\",\n"
+            + "                    \"name\": \"BytesOutPerSec\"\n"
+            + "                },\n"
+            + "                \"values\": [\n"
+            + "                    [\n"
+            + "                        1603301400,\n"
+            + "                        \"4096\"\n"
+            + "                    ],\n"
+            + "                    [\n"
+            + "                        1603301430,\n"
+            + "                        \"4096\"\n"
+            + "                    ]\n"
+            + "                ]\n"
+            + "            }\n"
+            + "        ]\n"
+            + "    }\n"
+            + "}", StandardCharsets.UTF_8);
+    }
+
+    private List<PrometheusQueryResult> expectedResults() {
+        return Arrays.asList(
+            new PrometheusQueryResult(
+                new PrometheusMetric(
+                    "b-1.test-cluster.org:11001",
+                    null, null),
+                Arrays.asList(
+                    new PrometheusValue(1603301400L, 1024),
+                    new PrometheusValue(1603301430L, 2048)
+                )
+            ),
+            new PrometheusQueryResult(
+                new PrometheusMetric(
+                    "b-2.test-cluster.org:11001",
+                    null, null),
+                Arrays.asList(
+                    new PrometheusValue(1603301400L, 4096),
+                    new PrometheusValue(1603301430L, 4096)
+                )
+            )
+        );
+    }
+
+    @Test(expected = IOException.class)
+    public void testFailureResponseWith200Code() throws Exception {
+        this.serverBootstrap.registerHandler("/api/v1/query_range", new HttpRequestHandler() {
+            @Override public void handle(HttpRequest request, HttpResponse response, HttpContext context) {
+                response.setStatusCode(200);
+                response.setEntity(new StringEntity(
+                    "{\"status\": \"failure\", \"data\": {\"result\": []}}", StandardCharsets.UTF_8));
+            }
+        });
+
+        HttpHost httpHost = this.start();
+        PrometheusAdapter prometheusAdapter
+            = new PrometheusAdapter(this.httpclient, httpHost, 30000);
```

Comment:
Nit: (applies to similar uses in this class) Can we move the hardcoded `30000` to a static variable?

Reply:
Done.